lib/index.ts

```diff
@@ -127,7 +129,7 @@ export function hasConflict(files: (File|FileSystemEntry|Node)[], content: Node[
 export function getConflicts<T extends File|FileSystemEntry|Node>(files: T[], content: Node[]): T[] {
 	const contentNames = content.map((node: Node) => node.basename)
 	const conflicts = files.filter((node: File|FileSystemEntry|Node) => {
-		const name = (node instanceof File || node instanceof FileSystemEntry) ? node.name : node.basename
+		const name = (node instanceof File || isFileSystemEntry(node)) ? node.name : node.basename
```

Comment:
Maybe better like this?

```suggestion
		const name = 'name' in node ? node.name : node.basename
```

Reply:
implemented